src/daft-csv/src/lib.rs

```diff
@@ -45,3 +52,13 @@ impl From<daft_io::Error> for Error {
         Error::IOError { source: err }
     }
 }
+
+type Result<T, E = Error> = std::result::Result<T, E>;
+
+#[cfg(feature = "python")]
+pub fn register_modules(_py: Python, parent: &PyModule) -> PyResult<()> {
```

Comment:
we should already have a `register_modules` in `src/daft-csv/src/python.rs`

Reply:
I was thinking that the module that defines them should also register them, hence the duplication, but I think what I'll do is I'll keep this top-level registerer and absorb the one in `src/daft-csv/src/python.rs` into this one, to keep the dependency direction nice.